.eslintrc.base.js

```diff
@@ -12,6 +12,9 @@ module.exports = {
   },
 
   rules: {
+    // re-enable once the proposed feature at https://github.com/gajus/eslint-plugin-jsdoc/pull/964#issuecomment-1936470252 is available
+    'jsdoc/check-line-alignment': 'off',
```

Comment:
So.... even if the proposed back compat feature is added, eslint-plugin-jsdoc has a bunch of other breaking changes that will cause problems.